engines/ospray/OSPRayModel.cpp

```diff
@@ -144,12 +144,13 @@ OSPGeometry& OSPRayModel::_createGeometry(GeometryMap& map,
                                           const char* name)
 {
     auto& geometry = map[materialId];
-    if (geometry)
-    {
-        ospRemoveGeometry(_primaryModel, geometry);
-        ospRelease(geometry);
-    }
-    geometry = ospNewGeometry(name);
+    // if (geometry)
```

Comment:
We leave this commented or we can remove it ?

Reply:
Removed, thanks!